articles/cognitive-services/QnAMaker/Overview/language-support.md

```diff
@@ -32,7 +32,7 @@ Consider the following:
 
 ## Supporting multiple languages in one QnA Maker resource
 
-This functionality is not supported in our current Generally Available (GA) stable release. Check out QnA Maker managed to test out this functionality. 
+This functionality is not supported in our current Generally Available (GA) stable release. Check out question answering(https://docs.microsoft.com/en-us/azure/cognitive-services/language-service/question-answering/overview) to test out this functionality. 
```

Comment:
Need to remove hard-coded en-us so that localization is not broken. Once this change is in place this PR can be merged.

```suggestion
This functionality is not supported in our current Generally Available (GA) stable release. Check out question answering(https://docs.microsoft.com/azure/cognitive-services/language-service/question-answering/overview) to test out this functionality. 
```
@jaydeepn sorry even though I am technically not the listed author anymore, when author is not listed in the metadata it fails back based on the docfx.json file in the root of azure-docs. I will modify the docfx.json file today so that these will go to Jeff in the future. These are small enough changes that if you make them against azure-docs-pr you have the ability to sign-off yourself regardless of the listed author. The changes will then still sync to azure-docs. Adding @jboback as an FYI